pkg/services/dns/dns_config_unix.go

```diff
@@ -0,0 +1,22 @@
+//go:build !windows
+
+package dns
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/miekg/dns"
+)
+
+func GetDNSHostAndPort() (string, string, error) {
+	conf, err := dns.ClientConfigFromFile("/etc/resolv.conf")
```

Comment:
It might be better to in future specify the location of this file as systems are moving away from using this; eg. systemd-resolvd

Reply:
For now, systemd-resolved provides a stub resolv.conf with a DNS resolver listening on 127.0.0.53